tests/functional/spec/features/submit_page/submit_page_summary.spec.js

```diff
@@ -0,0 +1,100 @@
+import DessertPage from "../../../generated_pages/summary/dessert.page.js";
+import DessertConfirmationPage from "../../../generated_pages/summary/dessert-confirmation.page";
+import NumbersPage from "../../../generated_pages/summary/numbers.page.js";
+import RadioPage from "../../../generated_pages/summary/radio.page.js";
+import SubmitPage from "../../../generated_pages/summary/submit.page.js";
+
+describe("Submit Page with Summary", () => {
+  beforeEach("Load the survey", () => {
+    browser.openQuestionnaire("test_summary.json");
+  });
+
+  it("Given a survey with a summary has been completed when the submit page is displayed, then it should contain a summary of all answers", () => {
```

Comment:
I realise this was copied from another spec file, but "survey" should be "questionnaire". Happy for a card to be raised to do this across everything.

Reply:
Ya I think we need to clean up as a whole as there is a lot of references to `survey` throughout the code base. But I have changed the ones in these spec files.